### ISSUE TITLE
🐛 CLI: yarn biome lint --only=accessibility and --skip=accessibility fails with "This group doesn't exist."

### ISSUE THREAD

**Issue by ckundo** [biomejs/biome#3700]
### Environment information

```block
CLI:
  Version:                      1.8.2
  Color support:                true

Platform:
  CPU Architecture:             aarch64
  OS:                           macos

Environment:
  BIOME_LOG_DIR:                unset
  NO_COLOR:                     unset
  TERM:                         "screen-256color"
  JS_RUNTIME_VERSION:           "v22.1.0"
  JS_RUNTIME_NAME:              "node"
  NODE_PACKAGE_MANAGER:         "yarn/1.22.19"

Biome Configuration:
  Status:                       Loaded successfully
  Formatter disabled:           false
  Linter disabled:              false
  Organize imports disabled:    false
  VCS disabled:                 true

Workspace:
  Open Documents:               0
```

### What happened?

```shell
% yarn --silent biome lint --only=accessibility fixtures/sample.tsx
```

and

```shell
% yarn --silent biome lint --skip=accessibility fixtures/sample.tsx
```

returns:

```shell
Error: couldn't parse accessibility: This group doesn't exist. Use the syntax `<group>/<rule>` to specify a
rule.
```

This happens with both `accessibility` and `accessibility/<rule>`.

### Expected result

The linter should run with only the accessibility rules applied.

### Code of Conduct

- [X] I agree to follow Biome's Code of Conduct

**Comment by ckundo** [comment]
I just figured this out. The documentation shows the group as named accessibility here https://biomejs.dev/linter/rules/#accessibility, but the group is labelled `a11y` in the code. Using `--only=a11y` works as expected.
